e2e/FR16.e2e-spec.ts

```diff
@@ -16,12 +16,17 @@ describe('Zonemaster test FR16 - [The advanced view should have a text describin
     expect(element(by.css('.alert.alert-info')).element(by.css('a')).getAttribute('fragment')).toBe('q12');
   });
 
-  it('should have a description text in multi languages', () => {
-    utils.setLang('en');
+  it('should have a description text in multi languages', async () => {
+    await utils.setLang('en');
     expect(element(by.css('.alert.alert-info')).element(by.css('a')).getText()).toContain('undelegated');
-    utils.setLang('fr');
+    expect(element(by.css('nav div.lang a.selected')).getAttribute('lang')).toBe('en');
```

Comment:
What does element() do when the locator matches multiple elements? In case it doesn't fail, maybe we should count the number of selected elements too, to make sure there is only one?

Reply:
It was only emitting a warning, I've updated the tests accordingly, and also moved them to FR05-* which seemed more appropriate.